templates/bonsai/contracts/tests/contract_tests/utils.rs

```diff
@@ -61,9 +62,9 @@ pub struct BonsaiMock {
 }
 
 impl BonsaiMock {
-    pub async fn spawn(
+    pub async fn spawn<Bytes: AsRef<[u8]> + Send + 'static>(
```

Comment:
Is `'static` what we want here or do we want to generalize `spawn` so that it takes a `'a`. I think this is probably fine for tests, but I can imagine scenarios where the registry might not last for the entire lifetime of the program.

Reply:
So a `'static` lifetime bound is required when passing data into a block for `tokio::spawn` (Since the thread is allowed to live for the lifetime of the program). This was new to me, so I learned more about it from https://tokio.rs/tokio/tutorial/spawning#static-bound

I think given the context, the `'static` lifetime bound is correct. This trait bound in particular can be satisfied by `&'static [u8]` or `Vec<u8>`. If the user has a buffer they want to reference that may not last until the end of the program, they can put it into an `Arc` and pass it in IIUC. At least with `tokio`, I think this is the the way of things.

Reply:
Thank you! This clears up some misconceptions I had in my head and now I see that we're just specifying a bound, not an annotation to the reference itself.